debian/cloud-init.postinst

```diff
@@ -276,6 +276,51 @@ fix_azure_upgrade_1611074() {
     rm "$tmpf" || :
 }
 
+fix_lp1889555() {
+    local oldver="$1" last_bad_ver="20.2-94-g3d06abc2-0ubuntu1"
```

Comment:
last_bad_ver should be the one in debian/changelog, in every series.

This is why on irc i was saying last_bad_ver being 20.2-95, assuming that all series will get 20.2-(at least 95)-gasfasasfd-...

Reply:
Right, per https://github.com/canonical/cloud-init/pull/514#discussion_r463724949, I have been approaching this as a packaging-only upload in each series, so the upstream version will not be bumped.

I'll update this to be series-appropriate in each series.